### PR TITLE
[MIG] sale_restricted_qty to 14.0

### PR DESCRIPTION
**Pull request by AshishHirapara**
Migrated sale_restricted_qty to 14.0 from 12.0

### CODE REVIEW

File: sale_restricted_qty/__manifest__.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Sale order min quantity",
-    "version": "12.0.2.0.0",
+    "version": "14.0.1.0.0",
     "category": "Sales Management",
-    "author": "Akretion, " "Odoo Community Association (OCA)",
-    "website": "https://github.com/OCA/sale-workflow/",
+    "author": "Ashish Hirpara, Akretion, Odoo Community Association (OCA)",
```
Comment (rousseldenis) [inline sale_restricted_qty/__manifest__.py]: @AshishHirapara Migration is not valid for changing authorship. Please add yourself to CONTRIBUTORS instead (same for headers).

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Add yourself to CONTRIBUTORS

File: sale_restricted_qty/__manifest__.py (hunk 1 above)
Comment (AshishHirapara) [reply sale_restricted_qty/__manifest__.py]: ok, I will update the changes. Thanks for the info.

**Comment by elvise** [comment]
@AshishHirapara @rousseldenis functional test ok!

**Review by elvise — APPROVED** [review]

**Review by francesco-ooops — APPROVED** [review]
LGTM (functional review)

@rousseldenis is it ok now?

File: sale_restricted_qty/__manifest__.py (hunk 1 above)
Comment (AshishHirapara) [reply sale_restricted_qty/__manifest__.py]: > @AshishHirapara Migration is not valid for changing authorship. Please add yourself to CONTRIBUTORS instead (same for headers).

@rousseldenis  This is updated.

**Comment by AshishHirapara** [comment]
> Add yourself to CONTRIBUTORS

@rousseldenis done

**Comment by elvise** [comment]
@rousseldenis what do you think can we merge ?

**Review by rousseldenis — APPROVED** [review]
@AshishHirapara For the next time, could you follow migration guide ?

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch

Especially the full process part in order to have a commit for sytnax fixing and then the migration commit. Thanks

**Comment by AshishHirapara** [comment]
> @AshishHirapara For the next time, could you follow migration guide ?
> 
> [OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch)
> 
> Especially the full process part in order to have a commit for sytnax fixing and then the migration commit. Thanks

@rousseldenis could you guide me on what went wrong with this commit Or which part of that guide did I miss? Because I have followed all the steps mentioned in that guide, and squashed previous commits of mine. So I will be more careful next time.
Thanks

**Comment by rousseldenis** [comment]
image

You should have two commits, one with "[IMP] sale_restricted_qty: black, isort, prettier" that should not be squashed and then the migration one. That in order to split the changes that are really migration

**Comment by alexey-pelykh** [comment]
@rousseldenis did you see that there's duplication in views for product variants?

**Pull request merged**